opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -145,8 +154,7 @@ def record_batch(
         match the key-value pairs in the label tuples.
 
         Args:
-            label_values: The values that will be matched against to record for
-                the handles under each metric that has those labels.
+            label_values: The values that will be matched against to record for.
```

Comment:
That's a mouthful. What about "The label values associated with all measurements in the batch."?

Also heads up that this line isn't wrapped.